code_generator/GdkCodeGenerator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -31,7 +31,12 @@ namespace <#= qualifiedNamespace #>
             private static readonly ComponentType targetComponentType = typeof(<#= componentDetails.TypeName #>);
 
             public override ComponentType[] ReplicationComponentTypes => replicationComponentTypes;
-            private static readonly ComponentType[] replicationComponentTypes = { typeof(<#= componentDetails.TypeName #>), typeof(Authoritative<<#= componentDetails.TypeName #>>), typeof(SpatialEntityId)};
+            private static readonly ComponentType[] replicationComponentTypes = { typeof(<#= componentDetails.TypeName #>), typeof(Authoritative<<#= componentDetails.TypeName #>>), 
+                typeof(SpatialEntityId) };
```

Comment:
Why has this been added to a new line?

Reply:
Must have been added accidentally and then copied over. Fixed back.